x-pack/plugins/apm/server/routes/settings/agent_configuration/__snapshots__/queries.test.ts.snap

```diff
@@ -1,88 +1,10 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`agent configuration queries findExactConfiguration find configuration by service.environment 1`] = `
-Object {
-  "body": Object {
-    "query": Object {
-      "bool": Object {
-        "filter": Array [
-          Object {
-            "bool": Object {
-              "must_not": Array [
-                Object {
-                  "exists": Object {
-                    "field": "service.name",
-                  },
-                },
-              ],
-            },
-          },
-          Object {
-            "term": Object {
-              "service.environment": "bar",
-            },
-          },
-        ],
-      },
-    },
-  },
-  "index": "myIndex",
-}
-`;
+exports[`agent configuration queries findExactConfiguration find configuration by service.environment 1`] = `undefined`;
```

Comment:
@MiriamAparicio I just noticed that a bunch of snapshots have been deleted. I don't think this was intended, was it?

Reply:
@sqren this might have happened when updating the snapshots. A lot of test were re-done or fixed, the snapshots update after, did you see any something wrong with these changes?

Reply:
The snapshots are supposed to ensure that the ES queries are correct. But now they are not doing anything (they are expecting `undefined`)

Reply:
Oh I see, it also happened in the custom link snapshots, I will open a ticket to look into it
Good catch